Vagrantfile

```diff
@@ -62,30 +63,29 @@ Vagrant.configure(2) do |config|
     puppet4master_config.vm.provision "shell", path: "scripts/install-puppet4master.sh"
     puppet4master_config.vm.provision "shell", path: "scripts/update-git.sh"
     puppet4master_config.vm.provision "shell", path: "scripts/install-vim-puppet-plugins.sh", privileged: false
-    # for some reason I have to restart network if host machine is a windows machine, but this needs more investigation
+
+    #Stop the default centos7 firewall service
     puppet4master_config.vm.provision "shell" do |remote_shell|
       remote_shell.inline = "systemctl stop firewalld"
-      remote_shell.inline = "systemctl disable firewalld"
```

Comment:
You need the disable line as well. Otherwise the firewall problem will come back when you do vagrant halt, followed by vagrant up.